src/Pages/index.js

```diff
@@ -0,0 +1,9 @@
+import {Home} from "./Home";
+import {History} from "./History";
+import {Login} from "./Login";
+import {Signup} from "./Signup";
+import {VideoList} from "./VideoList";
+import {WatchLater} from "./WatchLater";
+import {NoPageFound}from "./Error";
+
+export {Home,History,Login,Signup,VideoList,WatchLater,NoPageFound};
```

Comment:
We can use export directly instead of first importing and then exporting. 
For example `export {Home} from "./Home"` .